doc/How-Tos/Hosting/Hosting-setup.md

```diff
@@ -0,0 +1,33 @@
+# How to get started with hosting
+
+Hosting can be used to register services that will be accessible throughout the application via dependency injection. This tutorial will walk you through the critical pieces needed to leverage hosting in your application. 
+
+NOTE: If you used the `dotnet new` template to create the solution, some of the steps outlined here are unecessary. You can see whether your application already creates the IHost interface using the CreateDefaultBuilder method by looking for a generated `App.xaml.host.cs` file.
+
+## Step-by-steps
+1. Install the Uno.Extensions.Hosting library from NuGet, making sure you reference the package that is appropriate for the flavor of Uno in use.
+
+2. Set the LangVersion to leverage new C# features
```

Comment:
Remove this - it was included in my blog because I wanted to use implicit usings and some other c#10 features but not necessary here I think